docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,207 @@
+  >
+    <path
+      d="M15 11L13 9L17 9L15 11Z"
+      fill="#00814D"
+      stroke="#00814D"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M28.4893 15V12.9492H23.5332V11.835L27.8604 5.13574H29.6787V11.8691H31.0732V12.9492H29.6787V15H28.4893ZM24.75 11.8691H28.4893V6.18848H28.4482L24.75 11.8418V11.8691Z"
+      fill="#00814D"
+    />
+  </svg>
+  total number decreased or resolved in your branch
+- &nbsp;&#8239;<svg width="26" height="20" viewBox="0 0 26 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M13.9893 15V12.9492H9.0332V11.835L13.3604 5.13574H15.1787V11.8691H16.5732V12.9492H15.1787V15H13.9893ZM10.25 11.8691H13.9893V6.18848H13.9482L10.25 11.8418V11.8691Z" fill="#747994"/>
+</svg>
+&#8239;&#8239;&#8239;&#8239;total count, for example, <em>3 new and 1 existing</em>
+
+<!-- prettier-ignore-end -->
+
+<hr />
+
+The test state (failed, flaky, pending) is also indicated in the spec:
+
+- _new_ = the state was <u>not</u> previously captured, but now <u>is</u> captured (newly pending tests can imply an `it.skip()` was not removed)
+- _existing_ = the state <u>was</u> previously captured and now <u>is still</u> captured
+- _resolved_ = the state <u>was</u> previously captured but now is <u>no longer</u>
+
+:::
+
+### Review Test Comparison
+
+Navigating into the test detail view reveals a side-by-side comparison of the test results on both branches, as well as, artifacts so you can compare before vs. after your PR. This can help narrow the focus on changes introduced or resolved on either branch and elevate the source of flake within tests throughout different attempts. Review the test definition panel for diff snapshots to help quickly determine changes in test code.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/test2.png"
+  alt="Test Comparison View"
+/>
+
+## Github Pull Request Comments
+
+The Cypress Cloud Github Integration offers detailed test information via [PR comments](/guides/cloud/integrations/source-control/github#Pull-request-comments). Follow the "Review all test suite changes" link to load the Branch Review between the source and base branches.
+
+<DocsImage
+  src="/img/guides/cloud/integrations/github/pr-comment-fail.jpg"
+  alt="Cypress GitHub App PR"
+/>
+
+## Troubleshooting
+
+### Note on Available Data
+
+Branch Review is a powerful tool to compare two branches with recorded runs to Cypress Cloud. There are factors that can impact what is available to review between a source and base branch.
+
+Most importanly, unless a PR is open (not closed or merged), the branch review is not guaranteed to show you the latest run recorded for the branch. When a PR transitions into a closed/merged state, it does not consider any subsequent commits pushed to the branch as part of the PR itself. Be sure that the latest CI job has been triggered or you may see the _closest_ commit and not the one that was _just_ merged.
```

Comment:
1. typo: importanly
2. we typically are able to show the latest state for a PR even if it was closed or merged, as long as there were runs on the relevant commits for the PR
3. we don't use any "closest" commit logic for Branch Review – if a run is not found for master commit, we omit master data; if a run is not found for feat commit, we use the latest run on feat – we never try to do any location of commits that are "close" to the commits that we find

Reply:
I just want to make sure we're all aligned on #2 here. Has this logic changed since last week? From what I gather over slack, there could be some confusion over what might be showing in this scenario. The goal was to get ahead of seeing data that seems out of sync with the user's branches. 

cc @tbiethman

Reply:
@pstakoun for #3, what you describe is what I was getting at. Closest probably is misleading. I'll reword that.

Reply:
Opted to streamline this a bit as to not list specifics of edge cases. Especially with some logic updates on deck.